test/unit/uk/gov/hmrc/apidocumentation/controllers/DocumentationControllerSpec.scala

```diff
@@ -520,4 +570,36 @@ class DocumentationControllerSpec extends UnitSpec with MockitoSugar with ScalaF
       bodyOf(actualPage) should include regex s"aaa.*ddd.*www.*zzz"
     }
   }
+
+  "renderXmlApiDocumentation" must {
+    "when grouped documentation is disabled" should {
+      "return 404 not found" in new Setup(groupedDocumentationEnabled = false) {
+        theUserIsLoggedIn()
+
+        val result = underTest.renderXmlApiDocumentation("VAT")(request)
+
+        status(result) shouldBe NOT_FOUND
+      }
+    }
+
+    "when grouped documentation is enabled" should {
+      "render the XML API landing page when the XML API definition exists" in new Setup(groupedDocumentationEnabled = true) {
+        theUserIsLoggedIn()
```

Comment:
If you pull out Charities Online into a thisNameIsAGoodName val then it would show that the first test doesn't fail for the same reason as the last test.